### ISSUE TITLE
The Sparrow is a pretty worthless ship

### ISSUE THREAD

**Issue by Pointedstick** [endless-sky/endless-sky#316]
This came up in issue #309, but the Sparrow is basically useless for anything except becoming a pirate. It's not particularly fast or maneuverable for such a small ship, and its armament of two short-ranged guns is not going to win it any battles. Going up against any other armed ship is pretty much suicide. As a result, there is not much point in making it one of the three possible starting ships, as the only practical option it gives you is to become a pirate preying on unarmed merchant ships. But even then, it has very little cargo space so your looting opportunities are severely limited.

Bottom line: I think this ship could use some tweaking.

**Comment by gchapyzh** [comment]
Agreed. The gunboat or it's brother would make a much better beginner ship. Say, a turret, a single hardpoint with a meteor missile, lvl1 shields, some ion thrusters and a bit of cargo/ bunks.

**Comment by airtroop392** [comment]
The Sparrow, and for that matter, most of the "lower end" interceptors would be much better served as heavy fighters. Someone brought this up earlier, and I know that there was a guy on the Steam forums who already made it one, but really- the interceptors are too fragile for use by the player in any situation.

**Comment by gchapyzh** [comment]
Low level fighters should get some sort of weapon, something heavy, slow, something that shoots bolts of energy that deal large damage. This would give small ships a fighting chance against larger ships.

**Comment by Man-animal** [comment]
Sparrow is serviceable if you swap the crappy lasers for twin Meteor launchers or a single Javelin pod.

**Comment by airtroop392** [comment]
@Man-animal I guess it's just a matter of opinion. I tend to stay away from the more frail ships as I feel that if I were to ever enter in a fleet-level engagement with one of them, I would be utterly decimated by missiles, lasers, or otherwise.

**Comment by Man-animal** [comment]
They're not for fleet-level engagements, they're for low level combat related work; escort missions and small-time piracy/pirate-hunting.  They're relatively cheap, fast enough to escape most things they can't kill, and serve as natural upgrades for someone who starts in a Sparrow.

By the time you start getting into the heavier action you'll probably have upgraded to a proper warship, which seems to be intended.

**Comment by airtroop392** [comment]
I was referring to having interceptors in a fleet, which is why I mentioned a "fleet sized engagement", but looking back I can see that was pretty hard to understand. Whoops on my part.

**Comment by Man-animal** [comment]
Yeah, I'll agree that they (and fighters) have a lot of issues when being used as supporting ships.  A big part of that is the ubiquitousness of laser turrets and the lack of friendly fire.  I imagine you could still get some use out of them as cheap missile platforms, though.

**Comment by thenerdfreak** [comment]
I just swapped out the lasers for missiles, boarded and murdered a few other small interceptor ships and then i had enough to buy a light warship. It goes by faster than merchanting does (by a longshot)

**Comment by endless-sky** [comment]
Starting the game with a Sparrow is more challenging, but as others have noted, if you swap out one of its lasers for missiles you can capture pirate Sparrows easily and pay off your mortgage in a couple of weeks of game time.

Also, what wins in a fight: a single 20M credit capital ship, or a mob of 40 Sparrows (which cost the same amount)? Already, the only reason not to just fill your fleet with tiny ships (which give you more offensive capability per unit cost than capital ships) is the expense of losing them.

**Comment by gchapyzh** [comment]
@endless-sky kinda just sat there, figuring out how to respond to that, doing math and stuff.

40 Sparrows = 40 Crew = 160 lasers = 20 Million Credits.

In a fight, me and my 20 Million Heavy will DIE fighting 160 lasers. Even worse, 80 Missile launchers.

However, as I die, I will make sure to destroy 1/3 1/2 of the Sparrows. Also, sparrows have no chance of capturing me.

Pain of mental thought. Give me a moment to sit on this.

**Comment by airtroop392** [comment]
There's the risk and the reward- cost effectiveness and strength in number at the risk of a loss of investment. If you use 40-80 Sparrows, you're going to win a lot of battles, but you're also going to lose a lot of ships, which means you lose a little bit of money. Using heavy ships certainly puts more of a dent in your pocket, but they're well protected and less of a liability than the smaller interceptors.

**Comment by gchapyzh** [comment]
Issue here is probably that a sparrow cant really 1v1 any other ship. Every time I tried attacking something, I just got a missile up my ramp from a passing by frigate.

**Comment by airtroop392** [comment]
It can probably take on combat drones and fighters, but not much else. If you gave it some Hai shields and some missiles, it might be able to take on a Fury. But, I get your point.

**Comment by Man-animal** [comment]
A Sparrow can down a Hawk or one of the weaker light warships using nothing but Meteor missiles.

**Comment by gchapyzh** [comment]
Amm... yeah... A LOT of missiles...

**Comment by Man-animal** [comment]
Yeah, so?  A single looted heavy laser will pay for multiple full reloads, and ships that require more crew give more combat rating when disabled.  You have to work with what's available.

**Comment by endless-sky** [comment]
Given that the Sparrow is *supposed* to be the weakest combat ship you can fly, I don't think it needs any changes.

**Comment by eatsomedust4** [comment]
ngl here the sparrow is actually pretty insane for fighting other interceptors if your using it correctly and put it simply what you want to do with those is swap out the bulky battery with a supercapacitor and add 2 meteor missile pods and then fly the sparrow towards the enemy then launch missiles and then turn away from the enemy before your shields get damaged and if the enemy is using javelins then doge them until they run out and then the target becomes a sitting duck
